modal_proto/api.proto

```diff
@@ -1140,6 +1140,8 @@ message DictLenResponse {
 
 message DictListRequest {
   string environment_name = 1;
+  int32 max_objects = 2;
+  double created_before = 3;
```

Comment:
Is `max_objects` the amount of objects returned? Specifically:

```python
for vol in modal.Volume.objects.list(max_objects=20, created_before=0):
    assert len(vol) == 20
```

Reply:
Yes it's for pagination. Not sure I'll use those names exactly in the public API (also the `created_before` will be `datetime | str`).